lib/active_force/association/eager_load_builder_for_nested_includes.rb

```diff
@@ -0,0 +1,84 @@
+module ActiveForce
+
+  module Association
+    class InvalidAssociationError < StandardError; end
+
+    class EagerLoadBuilderForNestedIncludes
+
+      class << self
+        def build(relations, current_sobject, parent_association_field = nil)
+          new(relations, current_sobject, parent_association_field).projections
+        end
+      end
+
+      attr_reader :relations, :current_sobject, :association_mapping, :parent_association_field, :fields
+
+      def initialize(relations, current_sobject, parent_association_field = nil)
+        @relations = relations
+        @current_sobject = current_sobject
+        @association_mapping = {}
+        @parent_association_field = parent_association_field
+        @fields = []
+      end
+
+
+      def projections
+        [relations].flatten.each do |relation|
```

Comment:
Looks like `relations` isn't referenced anywhere else.  Could do `@relations = [relations].flatten` in the constructor instead of here to clean this up a bit.